ensembl/conf/ini-files/ovis_aries_rambouillet.ini

```diff
@@ -23,8 +23,8 @@ PHENOTYPE_PARAM   = 507
 PHENOTYPE_TEXT    = Coat colour, extension
 
 ENSEMBL_SOUND     = Bêêêêê! Bêêêêê!
-VEP_HGVS          = ENSOART00020000042.1:c.4A>T\nENSOART00020000993.1:c.75+16A>T\nENSOART00020030482.1:c.6del
-VEP_ID            = rs405918463\nrs402329723\nrs1091210299
-VEP_VCF           = 1 87437 . A T . . .\n2 20599 . A T . . .\n3 381158  . AC  A . . .
-VEP_SPDI          = NC_040252.1:87436:A:T\nNC_040253.1:20598:A:T\nNC_040254.1:381158:C: 
-VEP_ENSEMBL       = 1 87437 87437 A/T 1\n2 20599 20599 A/T 1\n3 381159  381159  C/- 1 
+VEP_HGVS          = ENSOART00020034561.2:c.4C>G\nENSOART00020035944.2:c.123+16G>T\nENSOART00020011412.2:c.5del
```

Comment:
the sandbox VEP failed -  http://wp-np2-11.ebi.ac.uk:6070/Ovis_aries_rambouillet/Tools/VEP/View?db=core;tl=T4O0z5dbvRM4igWd-53

It seems the cache files are missing for the job to run.

Reply:
Do you think this is fine and just a problem with cache?